src/components/themes/__tests__/themes-test-cases.js

```diff
@@ -81,4 +81,15 @@ testCases.custom = {
   }
 };
 
+testCases.pricing = {
+  description: 'pricing information',
+  element: (
+    <div>
+      {' '}
+      <Note theme="pricing">pricing note</Note>
+      <Tag theme="pricing" />
```

Comment:
@HeyStenson do you think you'll be using a pricing Tag? Just checking since some themes have a tag, other don't. If you do have use for a pricing tag, you'll also want to update the Tag component's propTypes to include "pricing" as a theme option.

Reply:
Oh hmmm, I hadn't considered this! But I think it could be useful - we could use it to differentiate pricing pages in the list of guides. I'll add this in!